doc/uftrace-record.md

```diff
@@ -512,6 +513,24 @@ all (supported) functions.  It shows two arguments for "main" and a single
 string argument for "puts".  If you simply want to see all arguments and
 return values of every functions (if supported), use -a/\--auto-args option.
 
+For example,
+
+    $ gcc -g -pg ../tests/s-abc.c -o abc
```

Comment:
I think that line 502~509's example is the better than this PR's example. 
Because example about `-a` option means that simply want to see all arguments and return values of every function about "puts" example.


> If you simply want to see all arguments and return values of every functions (if supported), use -a/\--auto-args option..

So I suggest the following example.

```sh
$ uftrace record -a . ./hello
Hello world

$ uftrace replay
# DURATION     TID     FUNCTION
....result......
```